### DISCUSSION TITLE
Icon request: icon-wedding-cake

### DISCUSSION THREAD

**Discussion by respectyoda** [FortAwesome/Font-Awesome#21156] [Icon Requests]
This icon would be applicable to wedding, photography, etc.
Here's some images of how the icon may look like:
![wedding-icon](https://cloud.githubusercontent.com/assets/3604751/2745390/5fec3b46-c735-11e3-9cbe-afdc512b3022.png)
![wedding-icon-1](https://cloud.githubusercontent.com/assets/3604751/2745391/5fef414c-c735-11e3-80b1-24fa2e541003.png)

**Comment by amy-frances** [comment]
+1

**Comment by jaepari** [comment]
+1

**Comment by tagliala** [comment]
Renamed as wedding cake, if you need wedding rings, please +1 #3025

**Comment by thatcatcancode** [comment]
+1

**Comment by JoeHana** [comment]
+1

**Comment by blazeworx** [comment]
+1 !!

**Comment by MayJow** [comment]
+1

**Comment by G-Driver** [comment]
+1
<img src="https://camo.githubusercontent.com/8045f7473457e2430c401e81c4c3f4631b2278ee/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f353735393336322f323232363039352f63636264346666362d396139622d313165332d383834612d6432396235656235393931642e706e67">

**Comment by tagliala** [comment]
@G-Driver this request is about a wedding cake, the birthday cake is already here :)

**Comment by G-Driver** [comment]
Yes I know but imagine it withoud the candle

**Comment by GwenLoguir** [comment]
+1 to wedding cake

**Comment by wildfireheart** [comment]
+1

**Comment by lyhil** [comment]
+1

**Comment by yvesrecour** [comment]
+1

**Comment by maryilewis** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by Crissov** [comment]
- 🎂 [U+1F382](http://emojipedia.org/birthday-cake/) Unicode emoji #222
- 🍰 [U+1F370](http://emojipedia.org/short-cake/)

**Comment by ezagroba** [comment]
+1 Wedding cakes don't have candles. I put fa-male and fa-female next to each other but it would be nice to have the single icon with the two rings.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
